snuba/consumers/consumer_builder.py

```diff
@@ -62,6 +63,28 @@ class MockParameters:
     std_deviation: int
 
 
+def validate_passed_slice(
+    storage_key: StorageKey, slice_id: Optional[int] = None
+) -> None:
+    # make sure this is a storage that is sliced
```

Comment:
should this be a docstring instead of a comment? also it was not obvious to me why 

```
assert slice_id < SLICED_STORAGES[storage_key.value]
```

was used, maybe you could say something like "make sure the slice id passed in is less than the number of total slices for the given storage" 
maybe its obvious for others tho

Reply:
That's a good point, I can write a docstring that clarifies why these validations are in place.